katalog/fluentd/fluent.conf

```diff
@@ -38,32 +46,3 @@
      retry_forever true
    </buffer>
 </match>
-
-<source>
-  @type prometheus
-</source>
-
-# input plugin that collects metrics from MonitorAgent
-<source>
-  @type prometheus_monitor
-  <labels>
-    host ${hostname}
-  </labels>
-</source>
-
-# input plugin that collects metrics for output plugin
-<source>
-  @type prometheus_output_monitor
-  <labels>
-    host ${hostname}
-  </labels>
-</source>
-
-# input plugin that collects metrics for in_tail plugin
-<source>
```

Comment:
I see some of the prometheus sources have been removed. Which where the reasons for their existence before (mostly a question for @lnovara ) and which are the reasons that brought to their removal ( @MiroslavVavra )

Reply:
@phisco I've never had a deep look at fluentd metrics but looking at https://github.com/fluent/fluent-plugin-prometheus, `prometheus_output_monitor` should be a superset of `prometheus_monitor` while `prometheus_tail_monitor` doesn't hold valuable metrics (at least from my POV).

If you want, I can deploy both and have a look at the precise differences.

In Fury, we are not relaying on any particular metric to be available as we just alert on the absence of the `up(job="fluentd")` metric.